docs/content/direct/deploy-on-k3d.md

```diff
@@ -0,0 +1,142 @@
+# Deploy KubeStellar on K3D
+
+This document shows how to deploy kubestellar v0.21.0 and a managed cluster on K3D
```

Comment:
Maybe drop the eversion?  
Maybe: ".. How to use K3D as the clusters for the hub and/or WECs for KubeStellar "

Reply:
Thanks, modified